packages/handlers/graphql/package.json

```diff
@@ -16,6 +16,9 @@
     "@graphql-mesh/types": "0.16.0",
     "@graphql-mesh/utils": "0.6.0",
     "fetchache": "0.0.4",
+    "@graphql-tools/code-file-loader": "6.2.5",
+    "@graphql-tools/graphql-file-loader": "6.2.4",
+    "@graphql-tools/load": "6.2.5",
```

Comment:
These packages already exist in the monorepo and versions were chosen to allow workspace hoisting.